webapp/channels/webpack.config.js

```diff
@@ -411,6 +411,8 @@ if (targetIsDevServer) {
                 '/api': {
                     ...proxyToServer,
                     ws: true,
+                    changeOrigin: true,
+                    secure: false,
```

Comment:
Just for my own knowledge, what are these changes for?

Reply:
Oh sorry, It's my mistake. That is the config for my local dev env.